frontendUiApi/src/main/java/com/virtuslab/gitmachete/frontend/ui/api/table/BaseGraphTable.java

```diff
@@ -11,8 +15,25 @@
 
 public abstract class BaseGraphTable extends JBTable {
   @UIEffect
+  @SuppressWarnings("nullness:method.invocation.invalid") // to allow for setAutoResizeMode despite the object isn't initialized yet
   protected BaseGraphTable(TableModel model) {
     super(model);
+
+    // Without set autoresize off column will expand to whole table width (will not fit the content size)
+    setAutoResizeMode(JBTable.AUTO_RESIZE_OFF);
```

Comment:
![image](https://user-images.githubusercontent.com/19799111/89778535-00f06100-db0e-11ea-8bc1-d9086ff97adb.png)

Why do we want it?

Reply:
Because without this tooltip will also appear when a mouse is hovering far away to the right in graph table (where there are no text)

Reply:
btw. brilliant branch names  :smile: :heart:

Reply:
> Because without this tooltip will also appear when a mouse is hovering far away to the right in graph table (where there are no text)

Maybe state that explicitly in the comment in line 22 (coz I didn't infer this from the current `column will expand to whole table width (will not fit the content size)`)